src/HttpMethod.php

```diff
@@ -4,16 +4,16 @@
 
 namespace SmartEmailing\Types;
 
-use SmartEmailing\Types\Comparable\ComparableInterface;
-use SmartEmailing\Types\Comparable\StringComparableTrait;
 use SmartEmailing\Types\ExtractableTraits\EnumExtractableTrait;
 
-final class HttpMethod extends Enum implements ToStringInterface, ComparableInterface
```

Comment:
Jen, proč se zrušila tan comparable traita?

Reply:
`\SmartEmailing\Types\Enum` ma svoji vlastni metodu `equals` a `HttpMethod` byl jediny Enum, ktery neco takoveho obsahoval a ta traita ji preposivala.
Koukam do historie a daval jsi to tam ty, melo to tehda nejaky vetsi umysl?
U toho kde se to tehda vsude pridavalo mi to pripada jako neumyslne pridani do tohoto enumu.